.github/workflows/codebuild-ci.yml

```diff
@@ -1,6 +1,11 @@
 name: AWS CodeBuild CI
 
 on:
+  # Manually trigger on specific branches
+  workflow_dispatch: 
+  push:
+    branches:
+      - dev
```

Comment:
Just for my understanding, why do we want to measure coverage on pushes to `dev`?
* On PRs, to understand if that PR is improving/reducing coverage
* On main, to give us our long term view.